deploy/bicep/groups/cloud-init-ubuntu.txt

```diff
@@ -7,6 +7,6 @@ runcmd:
   - sudo mkdir -p /tmp/downloadazcopy
   - sudo tar -xf /tmp/downloadazcopy-v10-linux --directory /tmp/downloadazcopy
   - sudo rm -f /usr/bin/azcopy
-  - sudo cp /tmp/downloadazcopy/azcopy_linux_amd64_*/azcopy /usr/bin/
+  - sudo cp /tmp/downloadazcopy/azcopy_linux_amd64_10.17.0/azcopy /usr/bin/
```

Comment:
Won’t this fail as soon as a new 10.x version of azcopy is released? If cloud-init isn’t doing the globbing, then maybe the command can be wrapped in a ‘sh -c’?

Reply:
Yes, it would fail. I was trying to find a way in their documentation without hard coding, but yeah `sh -c` should hopefully work. Thanks

Reply:
In general, I find the cloud-init documentation could have been more helpful. I don't know if there is an alternate way of testing, but whenever I make a change to the YAML file, I run it against a new multipass instance (as shown below), sometimes neither 'cloud-init.log' nor 'cloud-init-output.log' contains `runcmd` outcomes which makes it hard to troubleshoot. 

`multipass launch bionic --name test-vm8 --cloud-init Downloads/cloud-init.yaml`

Reply:
Yeah, I agree the docs could be a little better. Step 4 of [this doc](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/cloud-init-troubleshooting) also describes how you can find the downloaded cloud-init data. Not sure if that will be there or not if the file is "invalid" according to cloud-init.